base/setup/welcome/lang/hi-IN.rc

```diff
@@ -2,7 +2,7 @@
  * PROJECT:     ReactOS Welcome Hindi Translation
```

Comment:
Just "ReactOS Welcome" is enough....

Reply:
I thought to make it more particular at topic.

Reply:
The "PROJECT" field means the whole project and not just one file.

Reply:
So it is necessary to change it?

Reply:
Yes, it is necessary.

Reply:
Ok, I was just asking